spec/abstract-jobs.html

```diff
@@ -104,31 +104,40 @@ <h1>Objectives</h1>
       <h1>Liveness</h1>
 
       <p>
-        For some object _obj_, a <dfn>hypothetical WeakRef-oblivious</dfn>
-        execution with respect to _obj_ is an execution whereby
-        WeakRef.prototype.deref being called on a WeakRef whose referent is
-        _obj_ always returns *undefined*.
+        For some set of objects _S_, a <dfn>hypothetical WeakRef-oblivious</dfn>
+        execution with respect to _S_ is an execution whereby
+        WeakRef.prototype.deref being called on a WeakRef whose referent is an
+        element of _S_ always returns *undefined*.
       </p>
       <emu-note>
-        WeakRef-obliviousness, together with liveness, capture two
-        notions. One, that a WeakRef itself does not keep an object alive. Two,
-        that cycles in liveness does not imply that an object is live. To be
-        concrete, if determining _obj_'s liveness depends on determining the
-        liveness of another WeakRef referent, _obj2_, _obj2_'s liveness cannot
-        assume _obj_'s liveness, which would beg the question.
+        WeakRef-obliviousness, together with liveness, capture two notions. One,
+        that a WeakRef itself does not keep an object alive. Two, that cycles in
+        liveness does not imply that an object is live. To be concrete, if
+        determining _obj_'s liveness depends on determining the liveness of
+        another WeakRef referent, _obj2_, _obj2_'s liveness cannot assume
+        _obj_'s liveness, which would beg the question.
```

Comment:
“which would beg the question” isn’t clear to me (i realize this predates your PR)

Reply:
It's to point out a circularity. Determining _obj_'s liveness recursively cannot assume _obj_'s liveness. Is "would be circular" less confusing?

Reply:
Given that these paragraphs talk about how to resolve cycles, maybe "would be unresolvably circular" or something? I don't have a better concrete suggestion :-/

Reply:
Given that this is in a note, I think we can iterate on the wording after landing this PR. (All this stuff is pretty confusing IMO; I'm not really sure how to clarify it all; "beg the question" already gets at circularity and might be one of the easier parts.)